website/docs/providers/index.html.markdown

```diff
@@ -13,12 +13,56 @@ as physical machines, VMs, network switches, containers, and more. Almost any
 infrastructure type can be represented as a resource in Terraform.
 
 A provider is responsible for understanding API interactions and exposing
-resources. Providers generally are an IaaS (e.g. Alibaba Cloud, AWS, GCP, Microsoft Azure,
-OpenStack), PaaS (e.g. Heroku), or SaaS services (e.g. Terraform Cloud,
-DNSimple, Cloudflare).
+resources. Most providers configure a specific infrastructure platform (either
+cloud or self-hosted). Providers can also offer local utilities for tasks like
+generating random numbers for unique resource names.
 
-Use the navigation to the left to find available providers by type or scroll
-down to see all providers.
+## Providers on the Terraform Registry
```

Comment:
This is an incredibly minor point that I hesitate to raise, but I'm mentioning it since it's a heading and will thus likely attract external fragment links once it's published: "_on_ the Terraform Registry" feels a little "off" to me, as I've typically seen/heard us say "_in_ the registry" elsewhere. :thinking:

Reply:
Good call, sometimes I have header text brainfarts. 👍🏼